mergify_cli/ci/scopes/cli.py

```diff
@@ -0,0 +1,228 @@
+from __future__ import annotations
+
+import json
+import os
+import pathlib
+import subprocess
+import typing
+
+import click
+import pydantic
+import yaml
+
+
+if typing.TYPE_CHECKING:
+    from collections import abc
+
+
+SCOPE_PREFIX = "scope_"
+SCOPE_NAME_RE = r"^[A-Za-z0-9_-]+$"
+
+
+class ConfigInvalidError(Exception):
+    pass
+
+
+class ScopeConfig(pydantic.BaseModel):
+    include: tuple[str, ...] = pydantic.Field(default_factory=tuple)
+    exclude: tuple[str, ...] = pydantic.Field(default_factory=tuple)
+
+
+ScopeName = typing.Annotated[
+    str,
+    pydantic.StringConstraints(pattern=SCOPE_NAME_RE, min_length=1),
+]
+
+
+class Config(pydantic.RootModel[dict[str, ScopeConfig]]):
+    root: dict[ScopeName, ScopeConfig]
```

Comment:
You cannot have a configuration file the is named `mergify-ci` and that only handles scopes.
Either the file is `mergify-scopes` either it's `mergify-ci` and has a scopes section.